### PR TITLE
Ski profile for planning routes (#5021)

### PR DESCRIPTION
**Pull request by mahlzahn**
See discussion thread in [#5021](https://github.com/osmandapp/Osmand/issues/5021).

### CODE REVIEW

File: obf_creation/rendering_types.xml [hunk 1]
```diff
@@ -5637,6 +5638,33 @@
...
+		<routing_type tag="piste:type" value="connection" mode="register" base="true"/>
+		<routing_type tag="piste:type" value="hike" mode="register" base="true"/>
+		<routing_type tag="piste:type" value="nordic" mode="register" base="true"/>
+		<routing_type tag="piste:type" value="skitour" mode="register" base="true"/>
+		<routing_type tag="piste:type" value="sled" mode="register" base="true"/>
+		<routing_type tag="piste:type" value="snow_park" mode="register" base="true"/>
+		<routing_type tag="piste:difficulty" mode="amend" base="true"/>
+		<routing_type tag="piste:grooming" mode="amend" base="true"/>
+		<routing_type tag="piste:oneway" mode="amend" base="true"/>
+		<routing_type tag="area" mode="amend" base="true"/>
```
Comment (xmd5a2) [inline obf_creation/rendering_types.xml]: Why add area=yes if you use it only to avoid it?

File: obf_creation/rendering_types.xml [hunk 2]
```diff
@@ -5637,6 +5638,33 @@
...
+		<routing_type tag="aerialway" value="station" mode="register" base="true"/>
+
+
+		<routing_type tag="piste:type" value="downhill" mode="register" base="true"/>
+		<routing_type tag="piste:type" value="connection" mode="register" base="true"/>
+		<routing_type tag="piste:type" value="hike" mode="register" base="true"/>
+		<routing_type tag="piste:type" value="nordic" mode="register" base="true"/>
+		<routing_type tag="piste:type" value="skitour" mode="register" base="true"/>
+		<routing_type tag="piste:type" value="sled" mode="register" base="true"/>
+		<routing_type tag="piste:type" value="snow_park" mode="register" base="true"/>
```
Comment (xmd5a2) [inline obf_creation/rendering_types.xml]: This block can be simplified to 
'''xml
<routing_type tag="piste:type" mode="register" base="true"/>
'''

File: rendering_styles/skimap.render.xml [hunk 3]
```diff
@@ -639,6 +640,7 @@
 		<switch cycle="true" point="false" objectType="3" order="5">
 			<case tag="piste:type" value="downhill"/>
 			<case tag="piste:type" value="skitour"/>
+            <case tag="piste:type" value="connection"/>
```
Comment (xmd5a2) [inline rendering_styles/skimap.render.xml]: Formatting

File: rendering_styles/skimap.render.xml [hunk 4]
```diff
@@ -904,6 +908,7 @@
 			<switch>
 				<switch>
 					<case tag="piste:type" value="nordic"/>
+            		<case tag="piste:type" value="connection"/>
```
Comment (xmd5a2) [inline rendering_styles/skimap.render.xml]: Formatting

**Comment by xmd5a2** [comment]
Todo: ski profile needs to be introduced in profile list

**Comment by mahlzahn** [comment]
> Todo: ski profile needs to be introduced in profile list

The pull request [Osmand/#6718](https://github.com/osmandapp/Osmand/pull/6718) should do this job, I think.

**Pull request merged**